RELEASE_NOTES.md

```diff
@@ -1,5 +1,11 @@
 # Release Notes
 
+## Changes in Transact 0.3.7
+
+* Update semver dependency from `0.9` to `1.0`.
```

Comment:
put code quotes around the dependency names

Reply:
updated